public/docs/ts/latest/guide/testing.jade

```diff
@@ -1419,7 +1498,10 @@ a#router-link-stub
   ### Stubbing the _RouterLink_
 
   The `RouterLinkStubDirective` contributes substantively to the test
+
 +makeExample('testing/ts/src/testing/router-stubs.ts', 'router-link', 'testing/router-stubs.ts (RouterLinkStubDirective)')(format='.')
+
```

Comment:
Ha, ok. Not sure how I did it (yes ok, probably by hitting enter, haha) but I will be more careful.